Add manual scaling definition with redirection link (#67)
spheronFdn/docs: pages/compute/scaling/manual.mdx

import { Callout } from "nextra-theme-docs";
import Navigation from "components/navigation";

# Manual Scaling

Spheron offers Manual Scaling to adjust the computational resources allocated to an instance. The users can make decisions
to increase or decrease the allocated resources, such as CPU, memory, or storage, to meet changing performance requirements.
This type of scaling provides users with direct control over resource adjustments, allowing for optimization based
on specific workload characteristics and immediate operational needs.

> Fill this form to gain early access to Manual Scaling.

Reviewer: ```suggestion
Please schedule a team call to gain early access to Manual Scaling.
```

> Please schedule a team call to gain early access to Manual Scaling.

Reviewer: below add this link - https://meetings.hubspot.com/prashant-maurya/scaling-demo

Author: <img width="661" alt="Screenshot 2023-08-22 at 15 40 47" src="https://github.com/spheronFdn/docs/assets/30815249/e244266d-9496-46c8-a63c-081e79dd3640">
